### PR TITLE
New adapter: Colombia

### PR DESCRIPTION
**Pull request by ColombiaOnline**
## Type of change

- [x] New adapter  
## Description of change
We have uses some conditions for banner sizes.

- contact email of the adapter’s maintainer
-  anil.singh1@timesinternet.in

### CODE REVIEW

**Comment by ColombiaOnline** [comment]
@musikele 
Thanks for quick response,
I have fresh commit and resolved all issues. please reviews changes and push the file in prebid repository asap .

File: test/spec/modules/colombiaBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        'creativeId': '6b958110-612c-4b03-b6a9-7436c9f746dc-1sk24',
+        'dealId': '',
+        'currency': 'USD',
+        'netRevenue': true,
+        'ttl': 3000,
+        'referrer': '',
+        'ad': '<div>This is test case for colombia adapter</div>'
+      }];
+      let result = spec.interpretResponse(serverResponse, bidRequest[0]);
+      expect(Object.keys(result)).to.deep.equal(Object.keys(expectedResponse));
```
Comment (musikele) [inline test/spec/modules/colombiaBidAdapter_spec.js:134]: When Running `npx gulp test`, I get this error: 

```
✖ should get the correct bid response
        HeadlessChrome 81.0.4044 (Mac OS X 10.15.4)
      TypeError: Cannot convert undefined or null to object
          at keys (<anonymous>)
          at Function.keys (node_modules/es5-shim/es5-shim.js:1145:24)
          at Context.<anonymous> (webpack:///test/spec/modules/colombiaBidAdapter_spec.js:134:21 <- test/test_index.js:113581:67)
```

File: test/spec/modules/colombiaBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+          'uid': '2c0b634db95a01',
+          'height': 0,
+          'crid': '',
+          'statusMessage': 'Bid returned empty or error response',
+          'width': 0,
+          'cpm': 0
+        }
+      };
+      let result = spec.interpretResponse(response, bidRequest[0]);
+      expect(result.length).to.equal(0);
```
Comment (musikele) [inline test/spec/modules/colombiaBidAdapter_spec.js:149]: When running `npx gulp test`, I get this other error: 

```
✖ handles empty bid response
        HeadlessChrome 81.0.4044 (Mac OS X 10.15.4)
      TypeError: Cannot read property 'length' of undefined
          at Context.<anonymous> (webpack:///test/spec/modules/colombiaBidAdapter_spec.js:149:21 <- test/test_index.js:113595:67)
```

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "prebid.js",
-  "version": "3.12.0",
+  "version": "3.17.0-pre",
```
[the remark this answers is not in the document]
Comment (ColombiaOnline) [reply package-lock.json:3]: @musikele 
I have reverted old version.if others changes please charge the old version.

**Comment by ColombiaOnline** [comment]
Hi @Fawke ,
Thanks for suggestions
I have resolved all issues and quick inline comments for your points.
**1:** Resolved all test cases errors 
**2:** MediaTypes- Property  sizes removed and add mediaType objects in colombiaBidAdapter.md file
**3:** Add supportedMediaTypes variable in colombiaBidAdapter.js
**4:** It's done to adjust the bid value considering the viewability factor of the adslots.
**5:** I have already created docs PR.Please find below path
https://github.com/ColombiaOnline/prebid.github.io/blob/master/dev-docs/bidders/colombia.md
**6:** Please again running the command gulp test 

Please reviews changes and push the files in prebid repository asap .

**Pull request merged**